src/dbos-executor.ts

```diff
@@ -423,8 +431,23 @@ export class DBOSExecutor {
 
     const runWorkflow = async () => {
       let result: R;
+      // Track concurrent executions of the WF
+      if (!this.workflowStats.has(wf.name)) {
+        this.workflowStats.set(wf.name, new WorkflowStats());
+      }
+      const wfs = this.workflowStats.get(wf.name)!;
+      wfs.currentlyExecuting++;
+      wfs.nStarted++;
```

Comment:
Is `nStarted` used anywhere? Another concern is we probably don't need to separately track nRunningGlobally and currentlyExecuting, because given a workflow UUID only one can execute no matter what -- what use case do you have in mind for tracking local runs?

Reply:
nStarted is currently a write-only statistic.  But I'd be surprised if nobody is ever curious about it.

This is a generic comment meant for this and others below: production systems deserve some level of monitoring, some kind of safety mechanisms, etc.  There are common problems that happen with scheduler systems, one is that the schedule isn't what the user wanted, and the other is that the scheduler jobs pile up.  The information being collected is a foundation for monitoring, and eventually controlling for, these potential problems.